docket/plugins/upload.py

```diff
@@ -0,0 +1,80 @@
+import crescent
+import hikari
+import yaml
+from yaml.scanner import ScannerError
+
+plugin = crescent.Plugin("commands")
+
+
+@plugin.include
+@crescent.catch_command(IndexError)
+async def handle_index_error(error: IndexError, ctx: crescent.Context) -> None:
+    await ctx.respond(
+        "When uploading messages with code blocks you must have 2 code blocks.",
+        ephemeral=True,
+    )
+
+
+@plugin.include
+@crescent.catch_command(ScannerError)
+async def handle_yaml_error(error: ScannerError, ctx: crescent.Context) -> None:
+    await ctx.respond(
+        "There was an error parsing your config:"
+        "```"
+        f"{error.problem_mark}\n{error.problem}"
+        "```",
+        ephemeral=True,
+    )
+
+
+@plugin.include
+@crescent.message_command
+async def upload(ctx: crescent.Context, message: hikari.Message) -> None:
+    if not message.content:
+        await ctx.respond(
+            "This message is empty or Docket can not see message content on"
+            "this server.",
+            ephemeral=True,
```

Comment:
It should never be the case that Docket can't see message content, as it's not a permission but rather an intent. Just say "Message is empty"